pyfar/dsp/__init__.py

```diff
@@ -34,6 +34,10 @@
     InterpolateSpectrum,
 )
 
+from .levels import (
```

Comment:
Lets discuss in the weekly where this and related pulls should be imported into pyfar. There is more stuff such as the L_eq and and L_E to which we might want to implement in the future.